tests/components/xiaomi_ble/test_events.py

```diff
@@ -0,0 +1,33 @@
+"""Test Xiaomi BLE events."""
+
+from homeassistant.components.xiaomi_ble.const import DOMAIN
+
+from . import make_advertisement
+
+from tests.common import MockConfigEntry
+from tests.components.bluetooth import inject_bluetooth_service_info_bleak
+
+
+async def test_xiaomi_motion_detected(hass):
+    """Make sure that a motion detected event is fired."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="DE:70:E8:B2:39:0C",
+    )
+    entry.add_to_hass(hass)
+
+    assert await hass.config_entries.async_setup(entry.entry_id)
+    await hass.async_block_till_done()
+
+    assert len(hass.states.async_all()) == 0
+
```

Comment:
Example to capture events
```
tests/components/hue/test_device_trigger_v2.py:    async_capture_events,
tests/components/hue/test_device_trigger_v2.py:    events = async_capture_events(hass, "hue_event")
```

Reply:
Thanks. Added the test

Reply:
This is likely mergable as is to add the event but you might want to wait until you are sure you will be able to finish the device trigger code in case anything with the event has to change if there is a chance it won't be done before the next release to avoid a breaking change in the event data

Reply:
Yes, I agree. I’ve already started looking into creating the device_trigger.py file. Will need some more time to figure that out, but I will add that in this PR.